LICENSE

```diff
@@ -1,4 +1,4 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+This work by Kyle R. Conway (https://kylerconway.com) is licensed under the Creative Commons Attribution-ShareAlike 4.0
```

Comment:
Hi Kyle,

Generally the license applies to the repository as a whole so we avoid attributing work to any one individual in the LICENSE file.  See the main libseccomp LICENSE file as an example:

* https://github.com/seccomp/libseccomp/blob/master/LICENSE

What we typically do to give credit to individuals is create a CREDITS file (not present in the repo currently because the repo is empty, feel free to add one otherwise I'll take care of that), not to mention the audit trail in the git log (this is the main reason why I asked you to send a patch or a pull-request, this way you get properly credited in the git history).  Examples of both can be found in the main libseccomp repo:

* https://github.com/seccomp/libseccomp/blob/master/CREDITS
* https://github.com/seccomp/libseccomp/commits/master
* https://github.com/seccomp/libseccomp/graphs/contributors

Hopefully this makes sense, if not please let me know.